common/lib/xmodule/xmodule/modulestore/tests/test_xml_importer.py

```diff
@@ -391,4 +391,4 @@ def test_import_static_file(self):
                 base_dir=base_dir
             )
             mock_file.assert_called_with(full_file_path, 'rb')
-            self.mocked_content_store.assert_called_once()
+            self.mocked_content_store.generate_thumbnail.assert_called_once()
```

Comment:
Invalid assertion name for mock 1.0.1, upgrading revealed that we were checking the wrong mock object.